### ISSUE TITLE
intercept null hypothesis

### ISSUE THREAD

**Issue by hmcmonagle** [QERM514/assistance#32]
In homework 2, question 2b, we learned that the null hypothesis of the "intercept" row in the summary() table is that the coefficient for intercept (beta_0, sometimes written as alpha) = 0. This implies that the null hypothesis is a line with its intercept going through the origin. However, the null hypothesis for each predictor is that the slope of the line of best fit between the predictor and response variable is 0, and I thought that the null hypothesis for the model in general is just a horizontal line through the mean of all the data points, which is not going to have an intercept of 0 (unless the mean happens to be zero). How do we resolve this discrepancy? Is there a fundamental concept I'm missing here? 

In other words, I don't totally understand why the null hypothesis for the intercept is intercept = 0, rather than intercept = mean. 

If the null hypothesis for the intercept is indeed that the intercept = 0, then what's the significance of the "intercept only" model, which would be a horizontal line that passes through the origin? I can understand that the predictor coefficients would be 0 if that predictor does not explain any of the variation in the response variable, but a horizontal line with y-intercept = 0 seems strange and meaningless in the context of any real-world example (e.g. why would soil nitrogen = 0 regardless of the predictors, in the null model?)

Hope this question makes sense!

**Comment by mdscheuerell** [comment]
Yes, @hmcmonagle, this Q totally makes sense. The answer relates to our discussion about a single F-test versus multiple t-tests.

You are indeed correct about your interpretation of the null hypotheses for the different beta_i. However, it seems you are thinking about the individual tests in a conditional manner. That is, our null for the intercept is "H0: B_0 = 0", which is different from "H0: B_0 = 0 | B_1 = ... = B_j = 0". So, yes, if all of the slopes are zero, then it's impossible for the intercept (now mean) to also be zero unless the mean of the response also equals zero.

Perhaps another way to think about it is if you fit two different models, one with and the other without an intercept, and compared them via a likelihood ratio test or an information criterion, your null hypothesis is also "H0: B_0 = 0", and you're not making any statement about B_1 through B_j.

**Comment by hmcmonagle** [comment]
Ah okay, that makes sense, thanks! I'll make sure when I consider the null hypothesis for intercept that I'm not also making assumptions about the predictor slopes being zero as well.
